### DISCUSSION TITLE
Version 2 CDN example

### DISCUSSION THREAD

**Discussion by leonardehrenfried** [graphql/graphiql#2570] [Q&A / Support] [answered]
I would like to try out version 2 in a CDN but it seems that the example in the repo has not been updated yet: https://github.com/graphql/graphiql/blob/graphiql-v2/examples/graphiql-cdn/index.html

I tried replacing its JS/CSS URLs with the V2 versions already deployed to npm/jsdelivr but that didn't work.

Does someone have an example at hand?

**Comment by jonathanawesome** [comment]
~~A gist with a [working example](https://gist.github.com/jonathanawesome/0d2e27f94aa1d782d676944a4d78edbc).~~

I take that back, the canary version doesn't include the primary changes for v2. We may not see a CDN version of the new UI until there's a release candidate. @thomasheyenbrock, is that correct?

**Reply by acao** [reply]
We are thinking of cutting an alpha release, but for now you can use the latest canary versions for the graphiql 2 PR!

**Comment by thomasheyenbrock** [comment, marked as answer]
Version 2 has since been released, that also means the CDN example shows the new version now.
